kubeflow/core/prototypes/all.jsonnet

```diff
@@ -7,7 +7,7 @@
 // @optionalParam disks string null Comma separated list of Google persistent disks to attach to jupyter environments.
 // @optionalParam cloud string null String identifying the cloud to customize the deployment for.
 // @optionalParam tfAmbassadorServiceType string ClusterIP The service type for the API Gateway.
-// @optionalParam tfJobImage string gcr.io/kubeflow-images-staging/tf_operator:v20180226-403 The image for the TfJob controller.
```

Comment:
we need to find better way for that. Can we use latest rather than fixed revision? If we need to quickly patch something this will not be optimal.

Reply:
I think we want to pin to a specific version. The whole point of cutting a release 0.1 is to provide users a fixed and reproducible environment. So if we were to push a new image and then users automatically started using that image because images were tagged to latest, then behavior would change for users in unexpected ways.

If users want to use "latest" and not pin then they can do that by overriding the image.

If we need to release a fix we would do this by cutting a new release; e.g. 0.1.1
So we would build a new TF operator image and publish it. We would then update the kubeflow components and then cut that as 0.1.1 and tell people to use the 0.1.1 tag and not the 0.1 tag to get the fix.

At least that's my current opinion about how it should work. The overall release process and compatibility guarantees are still being thought out, so its possible we will do something different.